app/services/katello/pulp3/content_view_version/export.rb

```diff
@@ -91,6 +91,12 @@ def destroy_exporter(exporter_href)
           api.exporter_api.delete(exporter_href)
         end
 
+        def validate_chunk_size(size)
```

Comment:
can't  you just call this  from 'validate!' method in the same class ?

Reply:
Also add a unit test to test this condition

Reply:
Add the condition test here -> https://github.com/Katello/katello/blob/master/test/services/katello/pulp3/content_view_version/export_test.rb#L50